content/libraries/rio.md

```diff
@@ -59,10 +59,10 @@ import RIO
 
 When you need functionality for other data types like `ByteString` and
 `Text`, we recommend importing qualified from the `RIO` version of the
-module using the recommending qualified name. This avoids the need for
-modifying your `package.yaml` file for each new usage, ensures that
-only safe functions are imported by default, and ensures consistency
-across `rio`-using projects. For example:
+module using the qualified name. This avoids the need for modifying
```

Comment:
I think the only real change here was removing the word "recommending." Could you instead replace it with the word "recommended"?

Reply:
Done.